src/Pad.js

```diff
@@ -1,132 +1,141 @@
-import React, { Component } from 'react';
-import './Pad.css';
+import React, { Component } from "react";
```

Comment:
hmm I don't know why I swapped for double quotes, I'll see if I can turn that off